src/main/java/sirius/tagliatelle/compiler/Compiler.java

```diff
@@ -518,12 +518,20 @@ public Expression parseExpression(boolean skipWhitespaces) {
      */
     private String consumeStaticBlock() {
         StringBuilder sb = new StringBuilder();
+        int numberOfOpenBlocks = 1;
+
         while (!reader.current().isEndOfInput()) {
             if (isAtEscapedAt()) {
                 sb.append(reader.consume().getValue());
                 reader.consume().getValue();
             } else {
-                if (isAtPotentialEndOfStaticBlock()) {
+                if (reader.current().is('{')) {
+                    numberOfOpenBlocks++;
+                } else if (reader.current().is('}')) {
+                    numberOfOpenBlocks--;
+                }
+
+                if (isAtPotentialEndOfStaticBlock(numberOfOpenBlocks)) {
```

Comment:
soll mer das net in zeile 532 packen ala if (numberOfOpenBlock == 0) { return sb.toString()) }